cocos/2d/framework/ui-skew.ts

```diff
@@ -0,0 +1,206 @@
+/*
+ Copyright (c) 2025 Xiamen Yaji Software Co., Ltd.
+
+ http://www.cocos.com
+
+ Permission is hereby granted, free of charge, to any person obtaining a copy
+ of this software and associated documentation files (the "Software"), to deal
+ in the Software without restriction, including without limitation the rights to
+ use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies
+ of the Software, and to permit persons to whom the Software is furnished to do so,
+ subject to the following conditions:
+
+ The above copyright notice and this permission notice shall be included in
+ all copies or substantial portions of the Software.
+
+ THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ THE SOFTWARE.
+*/
+
+import { JSB } from 'internal:constants';
+import { ccclass, disallowMultiple, executeInEditMode, menu, serializable, type } from 'cc.decorator';
+import { Component } from '../../scene-graph/component';
+import { cclegacy, IVec2Like, v2, Vec2 } from '../../core';
+import { NodeEventType, TransformBit } from '../../scene-graph';
+import { TRANSFORM_ON, Node } from '../../scene-graph/node';
+
+const tempVec2 = v2();
+
+@ccclass('cc.UISkew')
+@menu('UI/UISkew')
+@disallowMultiple
+@executeInEditMode
+export class UISkew extends Component {
+    @serializable
+    private _value: Vec2 = v2();
```

Comment:
How about rename it to `_skewValue` or `_skew`?

Reply:
It's the `value` defined in `UISkew` component, I thought `_value` is enough to describe it.
If we have to do that, `_skew` is shorter for serialization.

Reply:
Renamed to `_skew`.